ChangeLog

```diff
@@ -1,6 +1,106 @@
 
-### 3.8.1dev <- NOTE: the release version number will be 3.9.0 ###
+### 3.8.1dev <- NOTE: the release version number will be 3.8.2 ###
 
+- Client: Added Vibraphone to list of instruments (#2043, #2080, #2158).
+  (contributed by @DavidSavinkoff, @softins, photo by @vibraphon)
+
+- Client: On Windows, if no driver found while installing, the "Run Jamulus"
```

Comment:
Seeing as this is an installer thing, can we also put the Mac client install fix below this? It's the main thing for Mac users in this release.

Reply:
Oh wait it's merged - I'll do it myself :-)